compose.yaml

```diff
@@ -0,0 +1,35 @@
+name: "dradis-ce-production"
+
+services:
+  web:
+    # image: dradis/dradis-ce:latest
+    build:
+      context: .
+      platforms:
+        - linux/amd64
+      args:
+        # Build arguments:
+        SSL_CERT_DIR: ${SSL_CERT_DIR}
```

Comment:
This will be removed before merge since we want to pull the image from Docker hub instead

Reply:
But leaving it now for testing & since we'll want to use this for documenting how to build the image